src/pages/AnimationPage.js

```diff
@@ -14,7 +15,20 @@ export default function AnimationPage({exerciseDuration}) {
         <BackButton />
       </Link>
       <BreatheAnimation />
-      <CountDownTimer exerciseDuration={exerciseDuration} />
+      <StyledContainer>
+        <StyledWrapper>
+          <CountDownTimer exerciseDuration={exerciseDuration} />
+        </StyledWrapper>
+      </StyledContainer>
     </>
   );
 }
+
+const StyledContainer = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const StyledWrapper = styled.div`
+  min-width: 10.6rem;
+`;
```

Comment:
It should also work if you put the min-width to `StyledContainer` and delete `StyledWrapper`, shouldn't it?

Reply:
Also too many divs for me 🙃 I have to add the min-width to CountDownTimer.js though, to prevent the timer from jumping.